lisa/tools/nvmecli.py

```diff
@@ -18,6 +18,10 @@ class Nvmecli(Tool):
     __ns_management_attachement_support = "NS Management and Attachment Supported"
     # [1:1] : 0x1   Format NVM Supported
     __format_device_support = "Format NVM Supported"
+    # Higher nvme-cli command add a mandatory parameter `--block-size`
```

Comment:
It's not a "higher command", it's a "higher version". It would be great, if we know the version.